static/js/components/ChannelBreadcrumbs_test.js

```diff
@@ -17,7 +17,7 @@ describe("ChannelBreadcrumbs", () => {
     const [first, second] = wrapper.find(Link)
     assert.equal(first.props.to, "/")
     assert.equal(first.props.children, "Home")
-    assert.equal(second.props.to, `/channel/${channel.name}`)
+    assert.equal(second.props.to, `/c/${channel.name}`)
```

Comment:
I think for these tests, while we're here, it would be good to replace the template literals with calls to the `channelURL` function, so if we make changes in the future we don't have to fiddle with it everywhere

Reply:
(and in the other test files where similar changes were made too)